api/provider/aws/service_update.go

```diff
@@ -7,6 +7,10 @@ import (
 	"github.com/quintilesims/layer0/common/models"
 )
 
+// Update updates an ECS Service using the specified Update Service Request.
```

Comment:
[nit] Maybe to avoid repetition.
"Update the ECS Service using the specified Update Service Request"
or
"Update is used to update the ECS Service"

Reply:
I will adopt your changes so that it's not so repetitive. But for reference, I modeled my sentence after golang docs that I know have this awkward repetitiveness:
https://golang.org/pkg/fmt/#Scan
https://golang.org/pkg/net/#UnixConn.Close
https://golang.org/pkg/bufio/#Reader.Read
etc.

Reply:
Okay gotcha. Thanks.